Example/Podfile

```diff
@@ -1,4 +1,4 @@
-platform :ios, '8.0'
+platform :ios, '10.0'
```

Comment:
Not sure upping the dependency is worth increasing two major iOS versions though?

Reply:
SnapKit requires iOS 10.0+ for Swift 5.... 😂

Reply:
Oh it's just the sample app, that's fine :D